source/jormungandr/jormungandr/instance.py

```diff
@@ -156,6 +158,18 @@ def __init__(
 
         self.zmq_socket_type = zmq_socket_type
 
+        self.equipment_providers = {}
+
+        self.configure_equipment_details_providers(instance_equipment_providers)
+
+    def configure_equipment_details_providers(self, instance_equipment_providers):
+        for provider_key in instance_equipment_providers:
+            provider = equipment_provider_manager.get_providers(provider_key)
```

Comment:
minor: taking `equipment_provider_manager` as a parameter will ease testing later.
It's better to inject dependency than get them from globals.

Reply:
:heavy_check_mark: